lib/general.sh

```diff
@@ -547,6 +547,8 @@ prepare_host()
 	curl patchutils python liblz4-tool libpython2.7-dev linux-base swig libpython-dev aptly acl \
 	locales ncurses-base pixz dialog systemd-container udev lib32stdc++6 libc6-i386 lib32ncurses5 lib32tinfo5 \
 	bison libbison-dev flex libfl-dev cryptsetup gpgv1 gnupg1 cpio"
+    
+	which aria2c >/dev/null || hostdeps="$hostdeps aria2"
```

Comment:
No need to check here. Just add aria2 to host dependencies

Reply:
I have installed a newer version of aria2c which is static build.

Reply:
Ok, this is harmless. It can stay.